README.md

```diff
@@ -1,4 +1,23 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four other blocks, with which it is lying side to side.
+Currently, however, the module is a little bit bugged, which means, that the
+fences will not get connected correctly the moment they are placed. However,
+placing an arbitrary block right next to a wrongly placed fence and then mining
+it again should fix the issue. Another problem this module currently have, is
+that the ground right around fence is severely darkened. This module also misses
```

Comment:
The darkening of ground is most probably due to SSAO. Would be good if you mention it.
"is severely darkened if you have SSAO enabled in your graphics settings."

"This module also misses"
Replace `misses` with `lacks`.